.gitignore

```diff
@@ -88,12 +88,16 @@ qrc_*.cpp
 # Qt creator
 *.pro.user
 
+# NetBeans
```

Comment:
I'm very open to removing it (or other parts), but the precedence was basically the ignore of the Qt Creator project files.